docs/UserGuide.adoc

```diff
@@ -58,53 +58,53 @@ Format: `help`
 
 === Adding a person: `addperson`
 
-Adds a person to the address book +
-Format: `addperson n/NAME [p/PHONE] [em/EMAIL] [a/ADDRESS] [r/REMARK] [tag/TAG]... [c/ NUS_MOD_LINK]`
+Adds a person to the timebook +
+Format: `add-person -n NAME -e EMAIL [-c NUS_MOD_LINK]`
 
 Examples:
 
-* `addperson n/ Catherine em/ catherineCanCook@imail.com`
-* `addperson n/ John Doe em/ JohnDoe@imail.com c/ 23F6B742CE`
+* `addperson -n Catherine -em catherineCanCook@imail.com`
+* `addperson -n John Doe -em JohnDoe@imail.com -c 23F6B742CE`
 
 === Adding a person: `addgroup`
 
 Adds a group to the address book +
-Format: `addgroup g/GROUP_NAME [r/GROUP_REMARK]`
+Format: `addgroup -g GROUP_NAME [-r GROUP_REMARK]`
 
 Examples:
 
-* `addgroup g/CS2103 r/Project Group`
-* `addgroup g/Friends`
+* `addgroup -g CS2103 -r Project Group`
+* `addgroup -g Friends`
 
 === Listing all persons : `list`
 
-Shows a list of all persons in the address book. +
+Shows a list of all persons in the timebook. +
 Format: `list`
 
 === Editing a person : `editperson`
 
-Edits an existing person in the address book. +
-Format: `editperson ed/PERSON_NAME [n/NAME] [p/PHONE] [em/EMAIL] [a/ADDRESS] [r/REMARK] [tag/TAG]...`
+Edits an existing person in the timebook. +
+Format: `edit INDEX [-n NAME] [-p PHONE] [-em EMAIL] [-a ADDRESS] [-t TAG]...`
```

Comment:
I believe we are still using the `PERSON_NAME` to select the person to edit. Do we have plans to transition to an index instead?